### DISCUSSION TITLE
Welcome to EmployeeManagementSystem Discussions!

### DISCUSSION THREAD

**Discussion by fix3-3** [way3-3/EmployeeManagementSystem#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fix3-3** [comment]
What have you done yet?Did you complete all codes of the project?

**Reply by way3-3** [reply]
Yes .We've done Admin module.Others are working on other features

**Comment by arefinthecybersec** [comment]
HELLO PERA...

**Reply by fix3-3** [reply]
pera nay chill
